driver.go

```diff
@@ -73,9 +73,9 @@ func (d *Driver) SetConfigFromFlags(opts drivers.DriverOptions) error {
 	if d.TritonAccount == "" {
 		return fmt.Errorf("%s driver requires the --%saccount/%sACCOUNT option", driverName, flagPrefix, envPrefix)
 	}
-	if d.TritonKeyPath == "" {
-		return fmt.Errorf("%s driver requires the --%skey-path/%sKEY_PATH option", driverName, flagPrefix, envPrefix)
-	}
+	// if d.TritonKeyPath == "" {
```

Comment:
Why bother commenting it out?

Reply:
I agree, I can probably get rid of these.